ui/src/path.ts

```diff
@@ -50,14 +50,17 @@ export const projectSource = (
 
 export const parseProjectSourceLocation = (
   querystring: string,
-  defaultRevision: string
+  defaultPeerId: string,
```

Comment:
I think for now this makes sense but in the future it might be better to fetch these source-related `defaults` in `source.ts` instead of passing them around through `path`